docs/src/components/pricing/PricingList.tsx

```diff
@@ -52,7 +52,7 @@ const Plan = React.forwardRef<
           endIcon={<KeyboardArrowRightRounded />}
           sx={{ py: 1 }}
         >
-          Get Started
+          Get started
```

Comment:
@siriwatknp 

There's a catch here with the Pricing table on mobile view. The Community and Pro plan can't have different button labels because of the way the code is set up... On desktop, I've already let them different, Community being **Get started** and Pro being **Buy now**.

Reply:
fixed.